pyproject.toml

```diff
@@ -84,3 +84,5 @@ addopts = "--reuse-db"
 testpaths = "pgpubsub/tests"
 norecursedirs = ".venv"
 DJANGO_SETTINGS_MODULE = "settings"
+filterwarnings = ["ignore::DeprecationWarning:django.http.request:",
+    "ignore::DeprecationWarning:django.utils.encoding:"]
```

Comment:
Where do these come from?

Reply:
Without this this warning are shown:

```
==================================================================== warnings summary ====================================================================
../../Library/Caches/pypoetry/virtualenvs/django-pgpubsub-LmhTrfgS-py3.11/lib/python3.11/site-packages/django/http/request.py:1
  /Users/roman/Library/Caches/pypoetry/virtualenvs/django-pgpubsub-LmhTrfgS-py3.11/lib/python3.11/site-packages/django/http/request.py:1: DeprecationWarning: 'cgi' is deprecated and slated for removal in Python 3.13
    import cgi

../../Library/Caches/pypoetry/virtualenvs/django-pgpubsub-LmhTrfgS-py3.11/lib/python3.11/site-packages/django/utils/encoding.py:266
  /Users/roman/Library/Caches/pypoetry/virtualenvs/django-pgpubsub-LmhTrfgS-py3.11/lib/python3.11/site-packages/django/utils/encoding.py:266: DeprecationWarning: Use setlocale(), getencoding() and getlocale() instead
    encoding = locale.getdefaultlocale()[1] or 'ascii'

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
```
django 3.x uses some outdated API. I'm not sure about django 4.

I can remove this.

Reply:
Do you want me to remove this ignores or keep them?

Reply:
We can just keep them. 
Only comment that needs addressed is the one about docs, then I can approve